src/main/java/net/starschema/clouddb/jdbc/BQStatement.java

```diff
@@ -262,9 +270,9 @@ private ResultSet executeQueryHelper(String querySql, boolean unlimitedBillingBy
                 // application bandwidth.
                 Thread.sleep(500);
                 this.logger.debug("slept for 500" + "ms, querytimeout is: "
-                        + this.querytimeout + "s");
+                        + getQueryTimeout() + "ms");
             }
-            while (System.currentTimeMillis() - this.starttime <= (long) this.querytimeout * 1000);
+            while (System.currentTimeMillis() - this.starttime <= (long) getQueryTimeout());
```

Comment:
The `timeoutMs` is normally a query param for the [`getQueryResults` endpoint](https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/getQueryResults) but since we're polling the jobs table in this loop it made sense to me to change the condition here.